impeller/compiler/shader_lib/impeller/gradient.glsl

```diff
@@ -145,4 +145,30 @@ vec3 IPComputeFixedGradientValues(float t, float colors_length) {
   return vec3(lower_index, upper_index, scale);
 }
 
+vec4 IPComputeColorFromTables(vec4 colors[256],
```

Comment:
I don't think this is used any more as the call overhead swamped everything else. I'll delete when I finalize the code.